dags/roger/core.py

```diff
@@ -476,6 +491,16 @@ def __init__(self, biolink=None, config=None):
         if not config:
             config = get_config()
         self.config = config
+
+        # We need a temp director for the DiskGraphMerger
+        self.temp_directory = os.path.dirname(os.path.abspath(__file__)) + "/" + self.config.kgx.merge_db_temp_dir
```

Comment:
instead of defining the merge_db_temp_dir relative to core.py , i would suggest having a function in the Util class that would resolve relative to the data_root_dir, an example might be something like this (https://github.com/helxplatform/roger/blob/e375945ba27c501176321af46a3eac1ca1054bd6/dags/roger/core.py#L165) , that way in k8s enviroment we can just worry about having one data volume mount  at the data_root level and sub directories can be managed via the util class ?

Reply:
It is  actually defining it like this, but when I changed it I forgot to delete the existing line:

        self.temp_directory = Util.merge_path(self.config.kgx.merge_db_temp_dir)

This OK right?